plugins/become/ksusu.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    name: ksusu
+    short_description: Kerberos substitute (ksu) user followed by substitute user (su).
+    description:
+        - This become plugins allows your remote/login user to execute commands as another user by calling su through the ksu utility.
+    author:
+    - Ed Schaller (@schallee)
+    options:
+        become_user_ksu:
+            description: User you use with ksu to run su
+            default: root
+            ini:
+              - section: privilege_escalation
+                key: become_user_ksu
```

Comment:
Shoudn't this be
```suggestion
                key: become_user
```
? Same for several of the other configuration options below.

Reply:
So, there are two different user options here. One for which user to ksu to and the other for which user to su to. In general become_user refers to the user that will run the task which is not become_user_ksu. 

Which principles are able to ksu to an account is defined by the list in the target user's home directory/.k5login. The issue is that adding ~/.k5login for every user you may want to become to (eg: httpd/apache user for example) is impractical. Hence the need for this plugin as if you can ksu to root you can then su to the target user.

How can I explain this better in the documentation? I honestly I cannot think of a case where setting become_user_ksu to something other than root (or not specifying it which is what ksu does). I tend towards flexibility of being being able to specify it but would be ok dropping the option for clarity.

I have considered having become_user_ksu and become_user_su for clarity and make become_user basically an alias for become_user_su.